### PR TITLE
Logging overhaul

### PR DESCRIPTION
**Pull request by latenssi**

### CODE REVIEW

File: chain_events/listener.go [hunk 1]
```diff
@@ -48,15 +46,17 @@ func NewListener(
...
-	if logger == nil {
-		logger = log.New(os.Stdout, "[EVENT-POLLER] ", log.LstdFlags|log.Lshortfile)
-	}
 
 	listener := &Listener{
-		nil, make(chan bool),
-		logger, fc, db, getTypes,
-		maxDiff, interval, startingHeight,
-		nil,
+		ticker:         nil,
```
Comment (nanuuki) [inline chain_events/listener.go:51]: 👌

File: handlers/middleware.go [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
 )
 
 func UseCors(h http.Handler) http.Handler {
 	return gorilla.CORS(gorilla.AllowedOrigins([]string{"*"}))(h)
 }
 
-func UseLogging(out io.Writer, h http.Handler) http.Handler {
-	return gorilla.CombinedLoggingHandler(out, h)
+func UseLogging(h http.Handler) http.Handler {
+	// TODO: log.New()
```
Comment (nanuuki) [inline handlers/middleware.go:15]: TODO comment still valid?

**Review by nanuuki — CHANGES_REQUESTED** [review]
`FLOW_WALLET_LOG_LEVEL` isn't documented; a mention of it in the README should suffice. Otherwise LGTM :)

File: handlers/middleware.go (hunk 2 above)
Comment (latenssi) [reply handlers/middleware.go:15]: Yes it is, thanks for pointing out :)

**Review by nanuuki — APPROVED** [review]

**Pull request merged**